contrib/openshift/manifests/manifests.yaml

```diff
@@ -0,0 +1,314 @@
+          protocol: TCP
+          port: 80
+          targetPort: ${{HTTP_TRANSPORT_PORT}}
+        - name: introspection
+          protocol: TCP
+          port: 8089
+          targetPort: ${{INTROSPECTION_PORT}}
+      selector:
+        service: matcher
+        app: clair
+  #
+  # index_report route
+  #
+  - kind: Route
+    apiVersion: route.openshift.io/v1
+    metadata:
+      name: clair-index-report
+      namespace: clair
+      labels:
+        app: clair
+        component: indexer
+    spec:
+      host: clair.stage.quay.io
+      path: /api/v1/index_report
+      to:
+        kind: Service
+        name: clair-indexer
+        weight: 100
+      port:
+        targetPort: ${{INTROSPECTION_PORT}}
+      wildcardPolicy: None
+  #
+  # index_state route
+  #
+  - kind: Route
+    apiVersion: route.openshift.io/v1
+    metadata:
+      name: clair-index-state
+      namespace: clair
+      labels:
+        app: clair
+        component: indexer
+    spec:
+      host: clair.stage.quay.io
+      path: /api/v1/index_state
+      to:
+        kind: Service
+        name: clair-indexer
+        weight: 100
+      port:
+        targetPort: ${{INTROSPECTION_PORT}}
+      wildcardPolicy: None
+  #
+  # vulnerability_report route
+  #
+  - kind: Route
+    apiVersion: route.openshift.io/v1
+    metadata:
+      name: clair-vulnerability-report
+      namespace: clair
+      labels:
+        app: clair
+        component: matcher
+    spec:
+      host: clair.stage.quay.io
+      path: /api/v1/vulnerability_report
+      to:
+        kind: Service
+        name: clair-matcher
+        weight: 100
+      port:
+        targetPort: ${{INTROSPECTION_PORT}}
+      wildcardPolicy: None
+parameters:
+  #
+  # indexer params
+  #
+  - name: INDEXER_DEPLOYMENT_REPLICAS
+    value: "4"
+    displayName: the number of indexers deployed
+  - name: INDEXER_CPU_LIMITS
+    value: "4"
+    displayName: the indexer's cpu limits in vCPUs
+  - name: INDEXER_CPU_REQS
+    value: "2"
+    displayName: the indexer's cpu requests in vCPUs
+  - name: INDEXER_MEM_LIMITS
+    value: "8192Mi"
+    displayName: the indexer's memory limits in vCPUs
+  - name: INDEXER_MEM_REQS
+    value: "4096Mi"
+    displayName: the indexer's memory requests in vCPUs
+  #
+  # matcher params
+  #
+  - name: MATCHER_DEPLOYMENT_REPLICAS
+    value: "4"
+    displayName: the number of matchers deployed
+  - name: MATCHER_CPU_LIMITS
+    value: "4"
+    displayName: the matcher's cpu limits in vCPUs
+  - name: MATCHER_CPU_REQS
+    value: "2"
+    displayName: the matcher's cpu requests in vCPUs
+  - name: MATCHER_MEM_LIMITS
+    value: "8192Mi"
+    displayName: the matcher's memory limits in vCPUs
+  - name: MATCHER_MEM_REQS
+    value: "4096Mi"
+    displayName: the matcher's memory requests in vCPUs
+  #
+  # shared params
+  #
+  - name: CLAIR_SECRET
+    value: "clair"
+    displayName: the vault secret containing clair's config
+  - name: CLAIR_IMAGE
+    value: ""
```

Comment:
Wouldn't you want this to point to a default image?  Same for tag

Reply:
Let me take a look, i was under the impression these values do not get used.

Reply:
These should be default values.  Deploments can override, but it doesn't make sense imo for the image to be blank as a default.

Reply:
Got it, I'll fix